ios/Podfile

```diff
@@ -5,6 +5,8 @@ target 'Integreat' do
   # Uncomment the next line if you're using Swift or would like to use dynamic frameworks
   use_frameworks!
 
+  pod 'react-native-simple-toast', :path => '../node_modules/react-native-simple-toast'
```

Comment:
This should not be added in the Podfile.

The linking process should just modify the xcode files.